### DISCUSSION TITLE
posts/velite-library-introduction

### DISCUSSION THREAD

**Discussion by giscus** [witch-factory/witch-next-blog#32] [General]
# posts/velite-library-introduction

컨텐츠 추상화 레이어를 제공하며 contentlayer를 대체 가능한 velite

https://witch.work/posts/velite-library-introduction

**Comment by junseokku** [comment]
contentlayer로 블로그 만들고 있는데, velite로 만드는 게 더 좋아보이네요.
글이 읽기 편해서 잘 읽었습니다. 좋은 글 작성해주셔서 감사합니다 :smiley:

**Comment by havenothorn** [comment]
현재 블로그 제작 중인데 Velite로 해봐야겠어요! 추천 감사드려요!!!! 👍
